pwiz_tools/Skyline/Controls/Graphs/DetectionPlotData.cs

```diff
@@ -0,0 +1,416 @@
+﻿using pwiz.Skyline.Model;

```

Comment:
You should add the standard file header comment to this file. Make sure to change the Original author to your own name.